lib/sycamore/sycamore/docset.py

```diff
@@ -974,29 +980,51 @@ def llm_filter(
             new_field: The field that will be added to the DocSet with the outputs.
             prompt: LLM prompt.
             field: Document field to filter based on.
-            threshold: Cutoff that determines whether or not to keep document.
+            threshold: Cutoff that determines whether to keep document.
+            keep_none:  whether to keep records with a None value.
```

Comment:
Does this mean "keep records with a None value for the provided field" ?

Reply:
Yeah, let me clarify